sql/keys_test.go

```diff
@@ -15,25 +15,41 @@
 //
 // Author: Tamir Duberstein (tamird@gmail.com)
 
-package sql_test
+package sql
 
 import (
 	"testing"
 
 	"github.com/cockroachdb/cockroach/keys"
 	"github.com/cockroachdb/cockroach/proto"
-	"github.com/cockroachdb/cockroach/sql"
 	"github.com/cockroachdb/cockroach/util/leaktest"
 )
 
+func TestNormalizeName(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	testCases := []struct {
+		in, expected string
+	}{
+		{"HELLO", "hello"},                       // Lowercase is the norm
+		{"ıİ", "ii"},                             // Turkish/Azeri special cases
+		{"no\u0308rmalization", "nörmalization"}, // NFD -> NFC.
```

Comment:
Done, though I'm assuming you meant: use `\u00f6` on the right side. `\u00xf` isn't a valid unicode escape.